### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by sadatakhtar**
Had to revisit login issue.

### CODE REVIEW

File: src/routes/auth/controllers.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ export const login = async (req, res) => {
  
   try {
     let myPost = req.body;
-    let authenticated =  users.find(e => e.userName.toLowerCase().includes(myPost.userName));
+    let authenticated =  users.find(e => e.userName === myPost.userName);
```
Comment (Wolfjawan) [inline src/routes/auth/controllers.js]: What is myPost stand for?

**Review by Wolfjawan — CHANGES_REQUESTED** [review]

File: src/routes/auth/controllers.js (hunk 1 above)
Comment (sadatakhtar) [reply src/routes/auth/controllers.js]: myPost stands for the POST request coming in, Maybe I should have chosen a better more meaningful name

Comment (sadatakhtar) [reply src/routes/auth/controllers.js]: renamed myPost to postBody

**Review by Wolfjawan — APPROVED** [review]
LGTM 👍

File: src/routes/auth/controllers.js (hunk 1 above)
Comment (ISTANBULBEKLE) [reply src/routes/auth/controllers.js]: Hi Sadat, as much as I remember you had already revised this, hadn't you?

**Review by ISTANBULBEKLE — APPROVED** [review]

**Pull request merged**